doc/installing-a-development-environment.md

```diff
@@ -90,11 +90,15 @@ Verify your Node.js installation by running:
     node --version
     npm --version
 
+### Local repo setup
+
+Clone the [cordova-docs] github repo to the local folder
```

Comment:
1. github => GitHub
1. to a specific folder or a random folder one likes?
1. Misses dot at end of sentence.

Reply:
@janpio Hi Jan, the review comments are implemented and thanks for your time.